sites/fast-website/src/app/css/style.css

```diff
@@ -348,12 +352,12 @@ fast-badge::part(control) {
     color: var(--neutral-foreground-rest);
   }
 
-  .site-navigation.debounce .anchor-wrapper {
+  site-navigation.debounce .anchor-wrapper {
     transition-duration: 0s;
   }
 
   site-navigation.opened .anchor-wrapper {
-    transform: translateY(0);
+    transform: translateY(-100%);
```

Comment:
This is causing the nav to slide up from the bottom of the screen for me

Reply:
This has been resolved per your help through discord :)